internal/models/alert_rule.go

```diff
@@ -18,19 +18,31 @@ type AlertingRule struct {
 
 	ScheduledTaskID *string
 	ExecutionStatus AlertingRuleExecutionStatus
+	AlertDelay      *AlertingRuleAlertDelay
```

Comment:
Why do we need a pointer to struct? Can we just use something like `AlertDelay *float32`?

Reply:
I did it, this made more sense than what I had ;)